### PR TITLE
[Feature] Accessibility labels for dialogs

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* Add accessibility labels, hints and order for dialogs

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

## What to Check
Verify that the following are valid
* Test the code
* Use accessibility inspector and/or VoiceOver to cycle through dialogs and match with Figma design

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift [hunk 1]
```diff
@@ -4,25 +4,33 @@
...
-            self.link = ""
+            title = ""
+            body = ""
+            linkDisplay = ""
+            link = ""
         }
+        accessibilityLabel = title + body + linkDisplay
+        // UIKit workaround to update accessibility when focus should be changed and isModal shouldn't be set
+        // for a consistent behaviour @AccessibilityFocusState should be used when min supported version is iOS 15+
+        UIAccessibility.post(notification: .screenChanged,
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift:32]: This code will be moved from VM in a next PR

Comment (palatter) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift:32]: Sounds good!

**Review by palatter — APPROVED** [review]
Looks good and tested.

**Comment by palatter** [comment]
New changes look good!

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/ViewComponents/Error/ErrorInfoViewModel.swift [hunk 2]
```diff
@@ -5,10 +5,14 @@
 
 import Foundation
 import Combine
+import UIKit
```
Comment (kevinyulu) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/ViewComponents/Error/ErrorInfoViewModel.swift:8]: is this import needed?

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift [hunk 3]
```diff
@@ -4,25 +4,33 @@
 //
 
 import Foundation
+import UIKit
+import SwiftUI
```
Comment (AmyL219) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift:8]: Could we remove `UIKit` and `SwiftUI` import here? Our viewModels layer don't reply any UIFramework though

**Review by kevinyulu — APPROVED** [review]
tested the calling view buttons, the voice over are announced correctly. 
Also tested the error banner on setup screen. looks good

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift (hunk 3 above)
Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingViewComponent/Banner/BannerTextViewModel.swift]: Updated!

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/ViewComponents/Error/ErrorInfoViewModel.swift (hunk 2 above)
Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/ViewComponents/Error/ErrorInfoViewModel.swift]: Updated!

**Pull request merged**